### PR TITLE
Gh504

### PR DESCRIPTION
**Pull request by PhelypeOleinik**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` updated

### CODE REVIEW

File: base/ltclass.dtx [hunk 1]
```diff
@@ -33,7 +33,7 @@
 %<*driver>
 % \fi
 \ProvidesFile{ltclass.dtx}
-             [2021/03/05 v1.3z LaTeX Kernel (Class & Package Interface)]
+             [2021/03/27 v1.3aa LaTeX Kernel (Class & Package Interface)]
```
Comment (PhelypeOleinik) [inline base/ltclass.dtx:36]: What version should this be?  It's not really a big change to deserve an increase to 1.4, but we've run short of letters in the alphabet :-)

**Comment by dbitouze** [comment]
"afterapackageends" at line 500 of `base/doc/ltnews33.tex` :wink:

File: base/doc/ltnews33.tex [hunk 2]
```diff
@@ -490,6 +490,19 @@ \subsection{Extending \pkg{latexrelease} to declare an entire module}
...
 
 
+\subsection{Small fix for rolling back prior to 2020-02-02}
+
+Whereas the \pkg{latexrelease} package can usually emulate an older \LaTeX{}
+kernel without much problem, rolling back to before the 2020-02-02 release
+didn't work properly because the management of the \cs{ExplSyntaxOn/Off} status
+for packages cannot be removed by the rollback without messing up catcodes after
+an \pkg{expl3}-based package is loaded.  This has been fixed and now rollback is
+more careful not to leave \pkg{ExplSyntaxOn}afterapackageends.
```
Comment (FrankMittelbach) [inline base/doc/ltnews33.tex:500]: as already mentioned

File: base/ltclass.dtx (hunk 1 above)
Comment (FrankMittelbach) [reply base/ltclass.dtx:36]: Historically we just moved to 1.4a (regardless of the size of change) so I would continue doing that here too.

**Review by FrankMittelbach — APPROVED** [review]
minor suggestions

File: base/doc/ltnews33.tex (hunk 2 above)
Comment (PhelypeOleinik) [reply base/doc/ltnews33.tex]: That was supposed to be a small joke, but it seems it didn't catch on :)

Fixed this and the version number.

Comment (FrankMittelbach) [reply base/doc/ltnews33.tex]: I was wondering about that briefly but then I know we will get it corrected by Barbara for the TUB version anyway :-)

Comment (dbitouze) [reply base/doc/ltnews33.tex]: > That was supposed to be a small joke, but it seems it didn't catch on :)

I couldn't imagine a joke here :wink: But it is indeed, and a funny one! :smile:

Comment (FrankMittelbach) [reply base/doc/ltnews33.tex]: well mabye you leave it :-)

Comment (dbitouze) [reply base/doc/ltnews33.tex]: Without a smiley (or any other bell), high is the probability that people see this as a typo :smile:

Comment (FrankMittelbach) [reply base/doc/ltnews33.tex]: \footnote{\ldots with this result.}

**Pull request merged**